creator-node/src/utils/legacyUtils.js

```diff
@@ -5,17 +5,20 @@ const axios = require('axios')
 const spawn = require('child_process').spawn
 const stream = require('stream')
 const { promisify } = require('util')
-const pipeline = promisify(stream.pipeline)
-const { logger: genericLogger } = require('../logging.js')
 
+const { logger: genericLogger } = require('../logging')
+const asyncRetry = require('./asyncRetry')
 const models = require('../models')
 const redis = require('../redis')
 const config = require('../config')
 const { generateTimestampAndSignature } = require('../apiSigning')
 const { libs } = require('@audius/sdk')
+
+const pipeline = promisify(stream.pipeline)
 const LibsUtils = libs.Utils
 
 const THIRTY_MINUTES_IN_SECONDS = 60 * 30
+const EMPTY_FILE_CID = 'QmbFMke1KXqnYyBBWxB74N4c5SBnJMVAiMNRcGu6x1AwQH' // deterministic CID for a 0 byte, completely empty file
```

Comment:
nit - since this is now defined twice, mind exporting it from here and consuming inside fileManager

Reply:
i agree, i dont think it's a good idea to create this var twice. I tried importing from fileManager, but there's a circular dependency here since fileManager also imports utils 😢 

I was thinking of defining it in config.js, but this variable doesnt seem like it should be placed there. so, the quick fix for this was just to define it here

Reply:
for sure - utils should def not import from fileManager, also this is not a config, but a local constant. i think its correct to define it here in legacyUtils, but export it from here and have fileManager import it which should work since it already imports utils?

Reply:
hmmm ya I wonder if there's a good constants file to put that in. `utils/constants` could work but I wonder if this could be broken up into a module to be organized by feature